ext/ffi/ffi.c

```diff
@@ -2777,6 +2778,14 @@ static zend_function *zend_ffi_get_func(zend_object **obj, zend_string *name, co
 	 && (ZSTR_VAL(name)[2] == 'p' || ZSTR_VAL(name)[2] == 'P')
 	 && (ZSTR_VAL(name)[3] == 'e' || ZSTR_VAL(name)[3] == 'E')) {
 		return (zend_function*)&zend_ffi_type_fn;
+	} else if (ZSTR_LEN(name) == sizeof("hasSym") -1
+	 && (ZSTR_VAL(name)[0] == 'h' || ZSTR_VAL(name)[0] == 'H')
```

Comment:
isn't there a function/macro for comparison? do we really need CI supporth here?